### DISCUSSION TITLE
Display Layouts help

### DISCUSSION THREAD

**Discussion by Jmangle93** [nasa/openmct#4205] [Community Q&A] [answered]
Hi all,

Is there a walkthrough/documentation for the layout object type?  I've tried to emulate the Mars Weather Display Layout [here](https://github.com/nasa/openmct-demo/blob/master/data/demo-models.json), and I have changed my folder into a layout type but I receive `this.layoutItems is undefined` and `this.getStyleReceiver() is null` TypeErrors. I'd like to create a default dashboard with multiple objects..let me know if this is the right track and whether you may be able to help.

Thanks!

**Comment by akhenry** [comment, marked as answer]
Hi! There is a section on Layouts in [our user guide](file:///Users/akhenry/Downloads/Open_MCT_Users_Guide-1.pdf). There is a quick start guide on page 6, and a more detailed guide on page 69.

**Reply by Jmangle93** [reply]
Thanks for the reply! I can set up a default layout from the UI easily enough by saving off the json, but when I drag and drop my telemetry plots to a layout, they don't transfer to the layout as plots but as [name,value] pair objects. What am I missing to transfer plots, tables, etc to a layout as their original component? Here's a snip of what the layout looks like
![Screenshot from 2021-09-13 11-43-36](https://user-images.githubusercontent.com/29695632/133131525-d1c59ee0-5eca-4826-ba14-20d8469143d3.png)

**Reply by Jmangle93** [reply]
Completely passed over 'Changing the Display Format of Telemetry' in the User Guide.... pg 73 :)
